pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -33,16 +33,27 @@ import (
 )
 
 const (
-	OrchestrationStatusStarting      = "starting"
+	// OrchestrationStatusStarting denotes the OSD provisioning is beginning.
+	OrchestrationStatusStarting = "starting"
+	// OrchestrationStatusComputingDiff denotes the OSD provisioning is scanning for disks that can
+	// be provisioned.
```

Comment:
I wish that `computingDiff` would be more self-explanatory...

Reply:
I could change this to "scanningDisks" if you and @travisn don't think it would cause any issues for users.

Reply:
I don't even remember why we need the computingDiff state. Seems like there is no action for the operator to take at that point anyway.

Reply:
The only benefit I see is the ability for the Operator to report that via the `CephCluster` status. But we aren't doing that I don't believe.